pootle/templates/admin/dashboard.html

```diff
@@ -57,30 +57,19 @@
 
 <div id="depchecks" class="module" lang="{{ LANGUAGE_CODE }}">
   <div class="hd">
-    <h2>{% trans "Dependency Checks" %}</h2>
+    <h2>{% trans "System Checks" %}</h2>
   </div>
   <div class="bd">
-    <ul id="required" class="icons">
-      {% for dep in required %}
-      <li class="{{ dep.dependency }}"><i class="icon-{{ dep.state }}"></i> {{ dep.text }}</li>
-      {% endfor %}
+    <ul>
+    {% for check in checks %}
+      <li>
+        <i class="icon-{% if check.is_serious %}reject{% else %}warning{% endif %}"></i>
```

Comment:
Just wondering: when *serious* checks fail, is there a chance we ever see this UI? If so, you can use `icon-error` as it was before.